modules/ROOT/pages/batch-filters-and-batch-aggregator.adoc

```diff
@@ -82,62 +77,54 @@ See xref:batch-error-handling-faq.adoc[Handling Errors During Batch Job] for mor
 . Accept Policy
 . Accept Expression
 
-== Batch Aggregator
 
-You can use the batch aggregator scope to accumulate a subset of records from a batch step, and bulk upsert them to an external source or service. +
-For example, rather than upserting each lead (i.e., record) in a batch to Salesforce, you can configure a Batch Commit to accumulate, say, 200 records and then upsert all of them to Salesforce in one chunk.
+[[aggregator_bulk_update]]
+== Performing Bulk Uploads from a Batch Aggregator Scope
 
-[source,xml,linenums]
-----
-<batch:step name="Step2">
-	<batch:aggregator size="200">
-     <salesforce:create type="Lead" .../>
-	</batch:aggregator>
-</batch:step>
-----
+Aggregation is useful for bulk uploading one or more arrays of records to an external server, such as a database server. Within the Batch Aggregator scope, you can add an operation, such as a bulk update, insert, or upsert operation, to load the records in bulk, instead of running an operation separately on each record.
```

Comment:
```suggestion
Aggregation is useful for bulk uploading one or more arrays of records to an external server, such as a database server. Within the Batch Aggregator scope, you can add an operation, such as a bulk update, insert, or upsert operation, to load one or more arrays of records to a server, instead of running an operation separately on each record.
```